roles/contrail_node/tasks/main.yaml

```diff
@@ -3,6 +3,12 @@
   include_role:
     name: contrail_common
 
+#- name: Generate and copy etcd client certs to contrail nodes
```

Comment:
Unnecessary code?

Reply:
It is not used right now but support for this will be required in some customer deployments.
Point 3 of commit message specifies this.